content/6_Advanced/Comb_Sub.mdx

```diff
@@ -86,39 +86,62 @@ Similar to the editorial, define $\texttt{dp[x][0][g]}$ to be the minimum cost t
 <Spoiler title="Code">
 
 ```cpp
-int n,b,c[5001],d[5001];
-vl dp[5001][2];
-vi child[5001];
-
-vl comb(vl a, vl b) {
-	vl c(sz(a)+sz(b)-1,INF);
-	F0R(i,sz(a)) F0R(j,sz(b)) ckmin(c[i+j],a[i]+b[j]);
-	return c;
+constexpr int MAX_GOODS = 5000;
```

Comment:
make code compile on its own

Reply:
So just add all of the `#include`s and that stuff?

Reply:
yea